dif-endorsed/README.md

```diff
@@ -0,0 +1,49 @@
+# DIF-endorsed DID Methods
+
+This page describes a process for DID methods to achieve a status of "DIF-endorsed DID Method", which is
```

Comment:
I propose a softer term for what this describes, e.g.:

- "DIF-qualified DID Method"
- "DIF-reviewed DID Method"
- "DIF-verified DID Method"
- "DIF-compliant DID Method"

Reply:
Great, thanks for these suggestions. I think it's important to agree on what we should call this. So let's discuss on the next call (tomorrow)!

Reply:
+1 to "DIF-reviewed" or "DIF-vetted" as terms rather than recommended.